sse2neon.h

```diff
@@ -6018,6 +6090,7 @@ FORCE_INLINE __m64 _mm_abs_pi8(__m64 a)
 // Concatenate 16-byte blocks in a and b into a 32-byte temporary result, shift
 // the result right by imm8 bytes, and store the low 16 bytes in dst.
 // https://www.intel.com/content/www/us/en/docs/intrinsics-guide/index.html#text=_mm_alignr_epi8
+#if defined(__GNUC__) && !defined(__clang__)
```

Comment:
Why does GCC need special treatment here?

Reply:
IIRC (I didn't actually write this particular bit, someone else did), the updated version didn't work in GCC